index.bs

```diff
@@ -3503,6 +3503,15 @@ the following algorithm returns <i>true</i>.
             <a href="#limit-bigint-numeric-overloading">a separate restriction on their use in
             overloading</a> below. Please also note <a href="#limit-bigint-numeric-unions">the
             advice about using unions of these types</a>.
+        1.  A [=callback function=] that does not have
+            [{{LegacyTreatNonObjectAsNull}}] extended attribute is
+            distinguishable from a dictionary-like category based on the
+            [=ECMAScript/callable=] state. When selecting a type for the
```

Comment:
Minor suggestion but: I think I'd avoid the phrasing “the callable state” because “state” implies “statefulness”? ES ensures/requires that “does/doesn’t implement [[Call]]” invariantly be an “eternal characteristic” of any ES value.

Reply:
Good call. Done.